.github/workflows/build-scan-push.yaml

```diff
@@ -62,6 +62,7 @@ jobs:
         uses: docker/setup-buildx-action@v1
 
       - name: Login to Docker Hub
+        if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
I suppose this condition is going to be true on non PR builds, but will not fail because `pull_request` is undefined ?

Reply:
That's what I think too 👍